partiql-eval/src/main/java/org/partiql/eval/value/Datum.java

```diff
@@ -547,6 +557,33 @@ static Datum int32Value(int value) {
         return new DatumInt(value);
     }
 
+    @Deprecated
+    @NotNull
+    static Datum intArbitrary(@NotNull BigInteger value) {
+        return new DatumBigInteger(value);
+    }
```

Comment:
This is mixing C-style names with SQL names. I say go for all SQL names for symmetry with PType. So update update int64Value to bigint.

Or visa-versa! No strong preference, just that right now they're mixed

Reply:
Yeah, I was tempted to change all of them to SQL as I ran through this PR, though I held back due to the diff. The intention is to use the SQL names as it's the public API. If you don't mind, my follow-up PRs aim to clean up some of our APIs.